plugin/case-reporting/src/main/java/org/opencds/cqf/ruler/casereporting/r4/MeasureDataProcessProvider.java

```diff
@@ -61,17 +56,12 @@ public Bundle extractLineListData(RequestDetails details,
 	private void gatherEicrs(IVersionSpecificBundleFactory bundleFactory, Map<String, Reference> populationSubjectListReferenceMap) {
 		Map<String, Bundle> eicrs = new HashMap<String, Bundle>();
 		for (Map.Entry<String, Reference> entry : populationSubjectListReferenceMap.entrySet()) {
-			// IQueryParameterType compositionReferenceParam = new ReferenceParam("subject", entry.getKey());
-			SearchParameterMap map = SearchParameterMap.newSynchronous();//.add("composition", compositionReferenceParam);
-			logger.warn(map.toNormalizedQueryString(measureResourceProvider.getContext()));
-			List<IBaseResource> bundles = myDaoRegistry.getResourceDao(Bundle.class).search(map).getAllResources();
-			for (IBaseResource baseBundle : bundles) {
-				if (baseBundle instanceof Bundle) {
-					Bundle bundle = (Bundle) baseBundle;
+			List<Bundle> bundles = search(Bundle.class, Searches.all()).getAllResourcesTyped();
```

Comment:
This line combines the Search API with the new TypedBundleProvider. Compare to the previous code.